packages/block-editor/src/components/fullscreen-mode/style.scss

```diff
@@ -10,27 +9,69 @@ body.js.is-fullscreen-mode {
 			height: calc(100% + #{ $admin-bar-height });
 		}
 
-		#adminmenumain,
+		// Hide the top adminbar toolbar.
```

Comment:
We might need to add and remove the `display: none` with JS (as adding it directly in the CSS might ruin the animation). The problem is that currently all the nav elements, and the skip links, are still focusable when hidden, and that causes some weirdness when tabbing through them:

<img width="219" alt="Screen Shot 2020-04-03 at 4 25 08 pm" src="https://user-images.githubusercontent.com/8096000/78327463-a9267400-75c8-11ea-832c-c0dc66d86e3d.png">

<img width="181" alt="Screen Shot 2020-04-03 at 4 24 54 pm" src="https://user-images.githubusercontent.com/8096000/78327471-af1c5500-75c8-11ea-814c-f2ee9a2cc5d2.png">

Reply:
Ah, good catch. I _think_ adding `pointer-events: none;` might work here. Using `display: none;`, like you said, will break the animation. I'll see what I can do to resolve this — thanks for taking a look.

Reply:
@tellthemachines + @shaunandrews Haiii! I just pushed something up that resolves this (I believe).

It hides the element wrapping the menu items. This allows for the outer (bar) to retain it's animation transition properties, but also prevents the inner links from being tabbable.